Makefile

```diff
@@ -27,7 +27,7 @@ pack-files:
 generate-proto: ## regenerate protos
 	@echo " > cloning protobuf from odpf/proton"
 	@rm -rf proton/
-	@git -c advice.detachedHead=false clone https://github.com/odpf/proton --depth 1 --quiet --branch main
```

Comment:
@arinda-arif Shouldn't this remain main?
@kushsharma

Reply:
yes, this will be changed to main once the proto is merged. I can mark this PR to draft to show the pre-requisite is not complete to avoid any confusion.